packages/@aws-cdk/aws-appmesh/lib/virtual-node.ts

```diff
@@ -1,13 +1,19 @@
 import * as cdk from '@aws-cdk/core';
+
 import { Construct } from 'constructs';
 import { CfnVirtualNode } from './appmesh.generated';
 import { IMesh, Mesh } from './mesh';
 import { ServiceDiscovery } from './service-discovery';
 import { AccessLog, BackendDefaults, Backend } from './shared-interfaces';
+import { renderTlsPolicy } from './tls-validation';
 import { VirtualNodeListener, VirtualNodeListenerConfig } from './virtual-node-listener';
 
 /**
- * Interface which all VirtualNode based classes must implement
+ * Interface which
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct as CoreConstruct } from '@aws-cdk/core'; all VirtualNode based classes must implement
```

Comment:
Not sure what happened here? Why this comment block here, with the import of Construct?

Reply:
Great catch! Originally, I attempted to add `render TlsPolicy` function to this class and in order to make it work I needed to import `Construct` from `@aws-cdk/core`. I thought I deleted the import statement but I must have missed it.